specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-01-01-preview/examples/watchlists/DeleteWatchlist.json

```diff
@@ -4,10 +4,20 @@
     "subscriptionId": "d0cfe6b2-9ac0-4464-9919-dccaee2e48c0",
     "resourceGroupName": "myRg",
     "workspaceName": "myWorkspace",
-    "watchlistAlias": "highValueAsset"
+    "operationalInsightsResourceProvider": "Microsoft.OperationalInsights",
+    "alias": "highValueAsset"
   },
   "responses": {
     "200": {},
+    "202": {
+      "body": {},
+      "headers": {
+        "Azure-AsyncOperation": {
+          "description": "Contains the status URL on which clients are expected to poll the status of the delete operation.",
```

Comment:
this should be a value, not definition.

example
headers: {
      azure-asyncoperation: "http://management........(url of the operation)"
}

Reply:
see other services for examples

Reply:
yep, sorry about that. Took a look at another example and updated it.